api/node/src/types/brush.rs

```diff
@@ -217,11 +241,52 @@ impl JsBrush {
     /// It is only implemented for solid color brushes.
     #[napi]
     pub fn to_string(&self) -> String {
-        if let Brush::SolidColor(_) = self.inner {
-            return self.color().to_string();
+        match &self.inner {
+            Brush::SolidColor(_) => {
+                return self.color().to_string();
+            }
+            Brush::LinearGradient(gradient) => {
+                return format!(
+                    "linear-gradient({}deg, {})",
+                    gradient.angle(),
+                    gradient_stops_to_string(gradient.stops())
+                );
+            }
+            Brush::RadialGradient(gradient) => {
+                return format!(
+                    "radial-gradient(circle, {})",
+                    gradient_stops_to_string(gradient.stops())
+                );
+            },
+            _ => String::default()
         }
+    }
+}
 
-        println!("toString() is not yet implemented for gradient brushes.");
-        String::default()
+fn gradient_stops_to_string<'a>(stops: impl Iterator<Item = &'a GradientStop>) -> String {
+    let stops: Vec<String> = stops
+        .map(|s| {
+            format!(
+                "rgba({},{},{}, {}) {}%",
```

Comment:
I suggest to have a space after each comma, not only the last one.

Reply:
done